src/test/resources/import.sql

```diff
@@ -1,3 +1,5 @@
+# noinspection SqlNoDataSourceInspectionForFile
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [syntax error at or near "#"](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)